### PR TITLE
Updated references in Identity Migration Doc

### PR DESCRIPTION
**Pull request by Tanato**
Updated references in Identity Migration to use the final namespaces of components:

    "Microsoft.AspNetCore.Identity.EntityFrameworkCore":  "1.0.0",
    "Microsoft.AspNetCore.Authentication.Cookies": "1.0.0",

Removed workaround to assure that database is created in Entity Framework Core

### CODE REVIEW

File: aspnet/migration/identity.rst [hunk 1]
```diff
@@ -14,14 +14,14 @@ In the previous article we :doc:`migrated configuration from an ASP.NET MVC proj
 Configure Identity and Membership
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In ASP.NET MVC, authentication and identity features are configured using ASP.NET Identity in Startup.Auth.cs and IdentityConfig.cs, located in the App_Start folder. In ASP.NET Core MVC, these features are configured in *Startup.cs*. Before pulling in the required services and configuring them, we should add the required dependencies to the project. Open *project.json* and add ``Microsoft.AspNetCore.Identity.EntityFramework`` and ``Microsoft.AspNetCore.Authentication.Cookies`` to the list of dependencies:
+In ASP.NET MVC, authentication and identity features are configured using ASP.NET Identity in Startup.Auth.cs and IdentityConfig.cs, located in the App_Start folder. In ASP.NET Core MVC, these features are configured in *Startup.cs*. Before pulling in the required services and configuring them, we should add the required dependencies to the project. Open *project.json* and add ``Microsoft.AspNetCore.Identity.EntityFrameworkCore`` and ``Microsoft.AspNetCore.Authentication.Cookies`` to the list of dependencies:
```
Comment (Rick-Anderson) [inline aspnet/migration/identity.rst]: Add  ``Microsoft.AspNetCore.Identity.EntityFrameworkCore`` and ``Microsoft.AspNetCore.Authentication.Cookies`` to the list of dependencies in *project.json*.

 ~~Before pulling in the required services and configuring them, we should add the required dependencies to the project. Open *project.json* and add ``Microsoft.AspNetCore.Identity.EntityFrameworkCore`` and ``Microsoft.AspNetCore.Authentication.Cookies`` to the list of dependencies:~~

Then delete the JSON  below.

File: aspnet/migration/identity.rst [hunk 2]
```diff
@@ -14,14 +14,14 @@ In the previous article we :doc:`migrated configuration from an ASP.NET MVC proj
...
 
 .. code-block:: none
```
Comment (Rick-Anderson) [inline aspnet/migration/identity.rst]: Delete this code block

**Comment by Tanato** [comment]
@Rick-Anderson done the requested changes. I took the liberty of wrap the line because seems to me that helps a little in readability.

**Comment by Rick-Anderson** [comment]
Thanks

**Pull request merged**